webapp/javascript/ui/Icons/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
```

Comment:
Honestly I would say to move this out of `ui/Icons`, since they have a different look and feel from the fontAwesome ones we use. So we probably don't want to have them reusable.